debug/doberman/main.go

```diff
@@ -58,6 +58,9 @@ func main() {
 
 	// write logo
 	tf, err := ioutil.TempFile("", "algorand-logo.png")
+	if err != nil {
+		panic(err)
```

Comment:
that's not so nice.. just print an error and os.Exit(1)

Reply:
I was following the convention of the other `if err != nil` handling in this program.